mne/source_tfr.py

```diff
@@ -0,0 +1,305 @@
+# -*- coding: utf-8 -*-
+#
+# Authors: Dirk Gütlin <dirk.guetlin@stud.sbg.ac.at>
+# TODO: This file uses a lot of stuff from io/base.py and source_estimate.py
+# TODO: Maybe name these persons as authors too?
```

Comment:
to me, the authors are the list of people that would know something about that file. It is the people I'm going to send them an email when things go south.

People who wrote the file or closely reviewed.